.github/workflows/ci.yml

```diff
@@ -138,13 +138,20 @@ jobs:
         env:
           SQL_NO_INLINE_SOURCEMAP: 1
 
-      - name: Run Core Unit Tests # {{SQL CARBON EDIT}} Rename to core for clarity
+      - name: Run Core Unit Tests # {{SQL CARBON EDIT}} Run core unit tests
```

Comment:
Don't change the comment - this is supposed to tell us why it was changed (which was because we were renaming it)